flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/PeriodicMaterializationManager.java

```diff
@@ -105,7 +116,7 @@ public void start() {
 
             LOG.info("Task {} starts periodic materialization", subtaskName);
 
-            scheduleNextMaterialization();
+            scheduleNextMaterialization(initialDelay);
```

Comment:
Shall we introduce this another method `scheduleInitialMaterization(initialDelay)` and make all calling of `scheduleNextMaterialization(0)` to `scheduleNextMaterialization()`? I think this looks better.